.github/workflows/deployment.yml

```diff
@@ -73,21 +73,21 @@ jobs:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
       - name: Defaults ✨
-        uses: azure/cli@v2.0.0
+        uses: azure/cli@v2
```

Comment:
could this not run the risk of updating to the latest version which could have breaking changes? my mysql?

Reply:
It shouldn't do as this is just Azure CLI version update, if there is a breaking change then pipelines will pick it up.